script/_common

```diff
@@ -16,8 +16,6 @@ function docker_build() {
 
 function docker_exec() {
   docker_build
-
-  appdir=$(cd $(dirname "$0")/../updater && pwd)
```

Comment:
I have a funny feeling that setting the `appdir` var isn't needed, but that the `cd $(dirname "$0")/../updater` is... 

Let's see what CI thinks, although that probably doesn't cover all workflows that use this...

Reply:
My understand is that this is to support running this script from outside of the `updater/` folder. Not sure if we need that.

Reply:
ah right, I missed that!